CMakeLists.txt

```diff
@@ -152,6 +152,30 @@ catkin_package(
 #   DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 # )
 
+# Mark executable scripts (Python etc.) for installation
+# in contrast to setup.py, you can choose the destination.
+#
+# We search for script files using the FILE(GLOB ...) macro, then assign the
+# result to openag_script_files.
+#
+# http://docs.ros.org/kinetic/api/catkin/html/howto/format2/installing_python.html
```

Comment:
I would love to make this a habit in our team. For any obtuse or dusty corner of the code, leave a note, any urls, a date and a name. That way if a dev truly gets stuck, they can reach out on GH, the forum or Slack to get unstuck.